src/ape/api/providers.py

```diff
@@ -6,10 +6,10 @@
 from hexbytes import HexBytes
 from web3 import Web3
 
-from ape.logging import logger
 from ape.types import TransactionSignature
 
 from ..exceptions import ProviderError
+from ..logging import logger
```

Comment:
just a little nit, but I usually don't like using double dot like this as it makes the code dependent on structure a little more

Reply:
Yeah, and I was actually wondering if you'd be interested in just going 100% absolute paths everywhere, I think `isort` created a whole new import section for relative imports rather than including them with the 1st party ones.